tests/everest/test_everest_initialization.py

```diff
@@ -28,3 +30,22 @@ def test_no_config_init():
 
     with pytest.raises(AttributeError):
         EverestRunModel.create("Frozen bananas")
+
+
+def test_site_config_with_substitutions(monkeypatch, copy_egg_test_data_to_tmp):
```

Comment:
Do you need the full egg case here? Should be enough to use the `min_config` fixture?